WordPress/src/main/res/values/strings.xml

```diff
@@ -1341,6 +1341,7 @@
     <string name="blog_name_invalid">Invalid site address</string>
     <string name="blog_title_invalid">Invalid site title</string>
     <string name="username_invalid">Invalid username</string>
+    <string name="checking_username">Checking username</string>
```

Comment:
> how about "user name" instead of "username"

Both calypso and wp-admin sign in screens use "username" so let's keep it as is for great justice (and consistency).
<img width="344" alt="screen shot 2017-03-03 at 11 54 25 am" src="https://cloud.githubusercontent.com/assets/1435271/23562500/4ef56286-0008-11e7-94b6-8fc7f0670d9b.png">
<img width="350" alt="screen shot 2017-03-03 at 11 54 18 am" src="https://cloud.githubusercontent.com/assets/1435271/23562504/50fbb22e-0008-11e7-82f2-5d44774994ca.png">